website/docs/clustering.md

```diff
@@ -51,8 +62,147 @@ NOTE: Clustering can only be scheduled for tables / partitions not receiving any
 ![Clustering example](/assets/images/blog/clustering/example_perf_improvement.png)
 _Figure: Illustrating query performance improvements by clustering_
 
-### Setting up clustering
-Inline clustering can be setup easily using spark dataframe options. See sample below
+## Clustering Usecases
+
+### Batching small files
+
+As mentioned in the intro, streaming ingestion generally results in smaller files in your data lake. But having a lot of
+such small files could bring down your query latency. From our experience supporting community users, there are quite a
```

Comment:
bring down query latency is confusing. I think what you meant was affect query latencies ? Or we can even go with higher query latencies ?

Reply:
Ah yes. Good catch Sudha! Will correct it.